configuration/defaults.go

```diff
@@ -122,16 +122,16 @@ vwIDAQAB
 	// defaultUserDeactivationInactivityPeriod the default number of days of inactivity before a user account can be deactivated
 	defaultUserDeactivationInactivityNotificationPeriod = 24 // 24 days
 	// defaultUserDeactivationInactivityPeriod the default number of days of inactivity before a user account can be deactivated
-	//	defaultUserDeactivationInactivityPeriod = defaultUserDeactivationInactivityNotificationPeriod + 7 // 31 days (7 days after defaultUserDeactivationInactivityNotificationPeriod)
-	defaultUserDeactivationInactivityPeriod = defaultUserDeactivationInactivityNotificationPeriod + 1 // 31 days (1 day after defaultUserDeactivationInactivityNotificationPeriod)
+	defaultUserDeactivationInactivityPeriod = defaultUserDeactivationInactivityNotificationPeriod + 7 // 31 days (7 days after defaultUserDeactivationInactivityNotificationPeriod)
 	// defaultPostDeactivationNotificationDelay the default number of milliseconds between to user account deactivation notifications by the same go routine
 	defaultPostDeactivationNotificationDelay = 500
-	// defaultCheServiceURL the default URL to the Che service
-	defaultCheServiceURL = "http://rhche-host:8080"
 	// defaultPostDeactivationNotificationDelay the default interval between 2 cycles of the user deactivation worker
 	defaultUserDeactivationWorkerIntervalSeconds = 60 * 60 // 1 hour
 	// defaultUserDeactivationNotificationWorkerIntervalSeconds the default interval between 2 cycles of the user deactivation notification worker
 	defaultUserDeactivationNotificationWorkerIntervalSeconds = 60 * 60 // 1 hour
 	// defaultUserDeactivationRescheduleDelayDays default of 1 day to re-attempt a failed deactivation attempt
 	defaultUserDeactivationRescheduleDelayHours = 24 // 24 hours
+
+	// defaultCheServiceURL the default URL to the Che service
+	defaultCheServiceURL = "http://rhche-host:8080"
```

Comment:
I believe this was not necessary (and we have something similar coming up in the e2e tests), but it's fine.

Reply:
I didn't add this config. I just moved it a few lines above for better defaults.go organization (to keep deactivation configuration closer to each other)

Reply:
ah ok, sorry. I misunderstood the change as it was presented here.